README.md

```diff
@@ -35,6 +35,10 @@ Listen podcasts anytime and anywhere to improve your listening skill. Podcasts o
 - [Luke’s English Podcast](http://teacherluke.co.uk/) - Luke is qualified english language teacher and stand-up comedian who provides engaging and entertaining podcast series.
 - [A Way with Words](http://www.waywordradio.org/category/episodes/) - Public radio program about language examined through history, culture, and family.
 - [This American Life](http://www.thisamericanlife.org/radio-archives) - One of the most popular podcast. Most of the stories are journalism, with an occasional comedy routine or essay.
+- [6 Minute English](http://www.bbc.co.uk/programmes/p02pc9tn/episodes/downloads) - Learn and practise useful English language for everyday situations with the BBC.
+- [CNN 10](http://edition.cnn.com/cnn10) - A 10-minute digital news show that explains global news to a global audience.
+- [Talk Python To Me](https://talkpython.fm/) - A podcast on Python and related technologies
```

Comment:
Please remove _Talk Python to Me_ since it's very specific to python developers.
The podcast should have more general theme to cover more broad audience

Reply:
OK, I'll remove that from this PR.

But I'm still thinking it might be useful for python developers. So, what about adding a category/sub-section such as **Developers** and/or **Python developers** or like that?

Reply:
Done

Reply:
@masayukig I think it's good idea to have sub section with good quality tech podcasts

Reply:
Thanks! I'm glad to hear that :) will do.